inc/Runner.php

```diff
@@ -12,7 +12,11 @@
 use WP_Filesystem_Base;
 
 /**
- * Project class.
+ * Runner class.
```

Comment:
This is just duplicating the file header, let's remove it? See also https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#2-classes.